src/ILAMB/ilamblib.py

```diff
@@ -1809,6 +1809,10 @@ def MakeComparable(ref,com,**keywords):
         # comparison, coarsen the comparison
         if np.log10(ref.dt/com.dt) > 0.5:
             com = com.coarsenInTime(ref.time_bnds,window=window)
+		# JITU: if reference time scale is significantly smaller than the 
+		# comprison, coarsen the reference
+        if np.log10(com.dt/ref.dt) > 0.5:
```

Comment:
I want to add a log entry here, philosophically this changes the reference which is somewhat against what we currently think